tests/pkcli/build_data/1-fconf.out/srv/host/v9.radia.run/etc/sysconfig/iptables

```diff
@@ -6,14 +6,12 @@
     -A INPUT -s 99.99.99.0/24 -j DROP
 -A INPUT -p icmp --icmp-type timestamp-request -j DROP
 -A INPUT -p icmp -j ACCEPT
-    -A INPUT -i em2 -p tcp -m state --state NEW -m tcp -s 192.168.1.1/24,127.0.0.1 --dport https -j ACCEPT
-    -A INPUT -i em2 -p tcp -m state --state NEW -m tcp --dport https -j DROP
-    -A INPUT -i em2 -p tcp -m state --state NEW -m tcp -s 192.168.1.1/24,127.0.0.1 --dport http -j ACCEPT
-    -A INPUT -i em2 -p tcp -m state --state NEW -m tcp --dport http -j DROP
     -A INPUT -i lo -j ACCEPT
     -A INPUT -i em1 -j ACCEPT
-    -A INPUT -i em2 -p tcp -m state --state NEW -m tcp --match multiport --dports 3100,3101,9999,http,https,pop3s,smtp,submission -j ACCEPT
 -A INPUT -i em2 -m state --state RELATED,ESTABLISHED -j ACCEPT
+    -A INPUT -i em2 -p tcp -m state --state NEW -m tcp --match multiport --dports 3100,3101,9999,http,pop3s,smtp,submission -j ACCEPT
```

Comment:
This still has http. Are we assuming nginx always redirects http to https?

Reply:
We don't allow anything over http except a redirect to https. For this case, we only need https blocked. This is simpler and clearer for now. Eventually we could support port ranges, but it's an unnecessary complication now.